src/audio/module_adapter/module/dts.c

```diff
@@ -316,10 +316,10 @@ static int dts_codec_apply_config(struct processing_module *mod)
 	config_header_size = sizeof(config->size) + sizeof(config->avail);
 	if (config->size < config_header_size) {
 		comp_err(dev, "dts_codec_apply_config() config->data is invalid");
-		return -EINVAL;
+		return 0;
```

Comment:
Should the comp_err() be changed to comp_warn() (or lower) if this is no longer an error? Do we lose the ability to flag real errors when we do not return -EINVAL?

Reply:
> Should the comp_err() be changed to comp_warn() (or lower) if this is no longer an error?

It sounds good. @joechengxperi please change to use comp_warn instead.

> Do we lose the ability to flag real errors when we do not return -EINVAL?

It should be fine to keep it silent. In my view the real error should not be met in common use cases (where the constant config is set by script on audio system server initializing). The checker after modification can still make sure that the invalid config won't be applied in any cases.

Reply:
@kv2019i @johnylin76 
Updated to comp_warn().